frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/RelationshipEdge/RelationshipEdge.tsx

```diff
@@ -41,6 +31,24 @@ export const RelationshipEdge: FC<Props> = ({
       <BaseEdge
         id={id}
         path={edgePath}
+        markerStart={
+          data?.cardinality
```

Comment:
This condition could be eliminated since cardinality is now always present!

Reply:
Oh, you're right! 
1a5a94ddb00c4260345de01386903c5a88447fc5